### PR TITLE
[New Rule] Executable File Creation with Multiple Extensions

### LINKED ISSUE #824: [New Rule] Executable File Creation with Multiple Extensions
## Description
Masquerading can allow an adversary to evade defenses and better blend in with the environment. One way it occurs is when the name or location of a file is manipulated as a means of tricking a user into executing what they think is a benign file type but is actually executable code.

## Required Info

- **Eventing Sources:**
`winlogbeat-*`, `logs-endpoint.events.*`

- **Platforms**
`windows`

## Optional Info
- **References:**

## Syntax
```
file where event.type == "creation" and 
  file.name:(
        "*.vbs.exe", 
	"*.vbe.exe", 
	"*.bat.exe",
	"*.js.exe",
	"*.cmd.exe",
	"*.wsh.exe",
	"*.ps1.exe",
	"*.pdf.exe",
	"*.docx.exe",
	"*.doc.exe",
	"*.xlsx.exe",
	"*.xls.exe",
	"*.pptx.exe",
	"*.ppt.exe",
	"*.txt.exe",
	"*.rtf.exe",
	"*.gif.exe",
	"*.jpg.exe",
	"*.png.exe",
	"*.bmp.exe",
	"*.hta.exe",
	"*.txt.exe",
	"*.img.exe"
	)
```

## Example Data

image

### PR DESCRIPTION
**Pull request by bm11100**
## Issues
resolves #824 

## Summary
Masquerading can allow an adversary to evade defenses and better blend in with the environment. One way it occurs is when the name or location of a file is manipulated as a means of tricking a user into executing what they think is a benign file type but is actually executable code.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/defense_evasion_file_creation_mult_extension.toml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+license = "Elastic License"
+name = "Executable File Creation with Multiple Extensions"
+risk_score = 47
+rule_id = "8b2b3a62-a598-4293-bc14-3d5fa22bb98f"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+file where event.type == "creation" and
```
Comment (rw-access) [inline rules/windows/defense_evasion_file_creation_mult_extension.toml:25]:
```suggestion
file where event.type == "creation" and file.extension : "exe" and
```

Comment (bm11100) [reply rules/windows/defense_evasion_file_creation_mult_extension.toml]: thanks, wasn't sure if that was redundant

File: rules/windows/defense_evasion_file_creation_mult_extension.toml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    "*.ppt.exe",
+    "*.txt.exe",
+    "*.rtf.exe",
+    "*.gif.exe",
+    "*.jpg.exe",
+    "*.png.exe",
+    "*.bmp.exe",
+    "*.hta.exe",
+    "*.txt.exe",
+    "*.img.exe"
```
Comment (threat-punter) [inline rules/windows/defense_evasion_file_creation_mult_extension.toml:49]:
```suggestion
    "*.img.exe",
    "*.iso.exe"
```

**Review by threat-punter — APPROVED** [review]
👌

File: rules/windows/defense_evasion_file_creation_mult_extension.toml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+name = "Executable File Creation with Multiple Extensions"
+risk_score = 47
+rule_id = "8b2b3a62-a598-4293-bc14-3d5fa22bb98f"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+file where event.type == "creation" and file.extension : "exe" and
+  file.name:(
```
Comment (Samirbous) [inline rules/windows/defense_evasion_file_creation_mult_extension.toml:26]: maybe u can add also same for macOS commonly abused extensions (.app and .pkg and .dmg): *.doc.app, *.pdf.app ... *.doc.pkg *.pdf.pkg *.doc.dmg

**Review by Samirbous — APPROVED** [review]

**Pull request merged**